lib/ansible/modules/cloud/vmware/vmware_export_ovf.py

```diff
@@ -300,9 +301,9 @@ def export_to_ovf_files(self, vm_obj):
                 ovf_descriptor_path = os.path.join(self.ovf_dir, ovf_descriptor_name + '.ovf')
                 sha256_hash = hashlib.sha256()
                 with open(self.mf_file, 'a') as mf_handle:
-                    with open(ovf_descriptor_path, 'wb') as handle:
+                    with open(ovf_descriptor_path, 'w') as handle:
                         handle.write(vm_descriptor)
-                        sha256_hash.update(vm_descriptor)
+                        sha256_hash.update(vm_descriptor.encode('utf-8'))
```

Comment:
Use `to_native` from `from ansible.module_utils._text import to_native`

Reply:
Since need bytes here and "to_native" will use "to_text" if it's python3, so I use "to_bytes" directly. Thanks.